docs/book/src/reference/version-dependencies.md

```diff
@@ -0,0 +1,15 @@
+# Versions dependency mapping
```

Comment:
This doc is meant for the developer for keeping the project up to dated with the possible latest versions.